pkg/reconciler/revision/reconcile_resources.go

```diff
@@ -202,22 +202,21 @@ func hasDeploymentTimedOut(deployment *appsv1.Deployment) bool {
 
 func (c *Reconciler) reconcileSecret(ctx context.Context, rev *v1.Revision) error {
 	ns := rev.Namespace
-	secretName := ns + "-" + networking.ServingCertName
 	logger := logging.FromContext(ctx)
-	logger.Info("Reconciling Secret: ", secretName)
+	logger.Info("Reconciling Secret: ", networking.ServingCertName, " at namespace: ", ns)
 
-	secret, err := c.kubeclient.CoreV1().Secrets(ns).Get(ctx, secretName, metav1.GetOptions{})
+	secret, err := c.kubeclient.CoreV1().Secrets(ns).Get(ctx, networking.ServingCertName, metav1.GetOptions{})
 	if apierrs.IsNotFound(err) {
 		namespace, err := c.kubeclient.CoreV1().Namespaces().Get(ctx, ns, metav1.GetOptions{})
 		if err != nil {
-			return fmt.Errorf("failed to get Namespace %q: %w", secretName, err)
+			return fmt.Errorf("failed to get Namespace %q: %w", ns, err)
 		}
 		if secret, err = c.createSecret(ctx, namespace); err != nil {
-			return fmt.Errorf("failed to create Secret %q: %w", secretName, err)
+			return fmt.Errorf("failed to create Secret %q at namespace %q: %w", networking.ServingCertName, ns, err)
```

Comment:
Knative project usually use `%s/%s` (namespace/name) for the log message like - 
https://github.com/knative/serving/blob/main/pkg/reconciler/nscert/nscert.go#L116